portal/templates/_macros/dashboard-tabs.njk

```diff
@@ -1,11 +1,17 @@
-{% macro nav(selected, id) %}
+{% macro nav(selected) %}
 
   {% set tabs = [
     {
-      id: 'deals',
+      id: 'bssDeals',
       href: '/dashboard',
-      text: 'Deal view'
-    }, {
+      text: 'BSS / EWCS Deal view'
+    },
+    {
+      id: 'gefDeals',
+      href: '/dashboard/gef',
+      text: 'GEF Deal view'
+    },
+    {
```

Comment:
Do we have an e2e test for clicking the tabs?

Reply:
I don't think so - we don't have any for the tabs at all in fact - transactions included. Want me to add them?

Reply:
Would be nice to do I think? Maybe get this merged first?

Reply:
Will add alongside the checker ticket